tests/cases/compiler/hugeDeclarationOutputGetsTruncatedWithError.ts

```diff
@@ -3,4 +3,4 @@ type props = "a" | "b" | "c" | "d" | "e" | "f" | "g" | "h" | "i" | "j" | "k" | "
 
 type manyprops = `${props}${props}`;
 
-export const c = null as any as {[K in manyprops]: {[K2 in manyprops]: `${K}.${K2}`}};
\ No newline at end of file
+export const c = () => null as any as {[K in manyprops]: {[K2 in manyprops]: `${K}.${K2}`}};
```

Comment:
Changed this to preserve the original intent of the test (test what happens when huge type nodes need to be printed).